### PR TITLE
feat(cli): add url option

### PR DESCRIPTION
**Pull request by a0viedo**

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -5,17 +5,26 @@ const program = require('../lib/program')
...
 const API_URL = process.env.API_URL
 const PROFILE_URL = process.env.PROFILE_URL
+const url = require('url')
 
 commander
-  .usage('--meetup-name [MEETUP_NAME] --event-id [EVENT_ID]')
+  .usage('--url [EVENT_URL] --meetup-name [MEETUP_NAME] --event-id [EVENT_ID]')
   .option('-m, --meetup-name <MEETUP_NAME>', `meetup's name. For example: banodejs`) // eslint-disable-line quotes
   .option('-e, --event-id <EVENT_ID>', `event's id. For example: 231888421`) // eslint-disable-line quotes
+  .option('-u, --url <EVENT_URL>', `event's url. For example:`) // eslint-disable-line quotes
```
Comment (durancristhian) [inline bin/cli.js:14]: For example? Have you forgot to paste the URL?

@a0viedo

**Comment by durancristhian** [comment]
Do you think that we should remove the other parameters? @a0viedo

File: bin/cli.js (hunk 1 above)
Comment (a0viedo) [inline bin/cli.js:14]: Good catch, yeah.

**Comment by a0viedo** [comment]
The other params are fine for me, but I often copy&paste just the URL.

**Pull request merged**